dcrtimed/backend/backend.go

```diff
@@ -14,15 +14,20 @@ import (
 )
 
 const (
-	ErrorOK         = 0 // Everything's cool
-	ErrorExists     = 1 // Digest exists
-	ErrorNotFound   = 2 // Generic not found error
-	ErrorNotAllowed = 3 // Generic not allowed error
+	// ErrorOK retuned when everything's cool
+	ErrorOK = 0
```

Comment:
If we haven't deployed V2 API to mainnet I'd like to make error 0 `invalid`. This was a mistake on my side and we should correct it for V2.

Reply:
Should I address that in another PR?

Reply:
Yes please.

Reply:
As discuss on matrix, we decided to bump the error code to 1, as it not stored and used to map to api specific error code.